astropy/io/ascii/latex.py

```diff
@@ -271,34 +291,65 @@ class Latex(core.BaseReader):
         Run the following code to see where each element of the
         dictionary is inserted in the LaTeX table::
 
-            from astropy.io import ascii
-            data = {'cola': [1,2], 'colb': [3,4]}
-            ascii.write(data, Writer=ascii.Latex, latexdict=ascii.latex.latexdicts['template'])
+            >>> from astropy.io import ascii
+            >>> data = {'cola': [1,2], 'colb': [3,4]}
+            >>> ascii.write(data, Writer=ascii.Latex, latexdict=ascii.latex.latexdicts['template'])
+            \begin{tabletype}[tablealign]
+            preamble
+            \caption{caption}
+            \begin{tabular}{col_align}
+            header_start
+            cola & colb \\
+             &  \\
+            header_end
+            data_start
+            1 & 3 \\
+            2 & 4 \\
+            data_end
+            \end{tabular}
+            tablefoot
+            \end{tabletype}
 
         Some table styles are predefined in the dictionary
         ``ascii.latex.latexdicts``. The following generates in table in
         style preferred by A&A and some other journals::
 
-            ascii.write(data, Writer=ascii.Latex, latexdict=ascii.latex.latexdicts['AA'])
+            >>> ascii.write(data, Writer=ascii.Latex,
+            ...             latexdict=ascii.latex.latexdicts['AA'])
+            \begin{table}
+            \begin{tabular}{cc}
+            \hline \hline
+            cola & colb \\
+            \hline
+            1 & 3 \\
+            2 & 4 \\
+            \hline
+            \end{tabular}
+            \end{table}
 
         As an example, this generates a table, which spans all columns
         and is centered on the page::
 
-            ascii.write(data, Writer=ascii.Latex, col_align='|lr|',
-                        latexdict={'preamble': r'\begin{center}',
-                                   'tablefoot': r'\end{center}',
-                                   'tabletype': 'table*'})
+            >>> ascii.write(data, Writer=ascii.Latex, col_align='|lr|',
+            ...             latexdict={'preamble': r'\begin{center}',
+            ...                        'tablefoot': r'\end{center}',
+            ...                        'tabletype': 'table*'})
+            \begin{table*}
+            \begin{center}
+            \begin{tabular}{|lr|}
+            cola & colb \\
+            1 & 3 \\
+            2 & 4 \\
+            \end{tabular}
+            \end{center}
+            \end{table*}
 
     **caption** : Set table caption
-        Shorthand for::
-
-            latexdict['caption'] = caption
+        Shorthand for:: ``latexdict['caption'] = caption``
 
     **col_align** : Set the column alignment.
         If not present this will be auto-generated for centered
-        columns. Shorthand for::
-
-            latexdict['col_align'] = col_align
+        columns. Shorthand for:: ``latexdict['col_align'] = col_align``
```

Comment:
I don't see the point of changing this. We can leave it as before without a doctest.

Reply:
Maybe needs the double back tick? I think it is a bit better, keeping the items shorter.